docker/rumble/Dockerfile

```diff
@@ -16,10 +15,10 @@ RUN yum install -y cern-hadoop-config spark-bin-3.0 hadoop-bin-2.7.5 git git-cor
 ENV PATH $PATH:/usr/hdp/hadoop-2.7.5/bin:/usr/hdp/spark3/bin
 
 WORKDIR $WDIR
-RUN curl -k -L -O https://golang.org/dl/go1.16.4.linux-amd64.tar.gz && \
-    tar xfz go1.16.4.linux-amd64.tar.gz && \
+RUN curl -k -L -O https://golang.org/dl/go1.16.5.linux-amd64.tar.gz && \
```

Comment:
you should not download go tarball, instead start from go docker image like das-server, see https://github.com/dmwm/CMSKubernetes/blob/master/docker/das-server/Dockerfile#L1